src/Kerbero.Common/Repositories/IHttpNukiClient.cs

```diff
@@ -0,0 +1,8 @@
+using Kerbero.Common.Entities;
+
+namespace Kerbero.Common.Repositories;
+
+public interface IHttpNukiClient
```

Comment:
i don't get the idea behind di client @salvanicola

Reply:
The business logic needs an httpclient to call the nuki API, but he can't talk with the layers below. So he asks to the infrastructure object, which we'll call him to give him an implemented httpclient.